internal/evaluator/conftest_evaluator.go

```diff
@@ -49,21 +49,20 @@ type ConfigurationPaths struct {
 }
 
 // NewConftestEvaluator returns a properly initialized ConftestEvaluator for usage
```

Comment:
Let's update the comment with the correct capitalization of `conftestEvaluator`.